p2p_wallet/Scenes/Main/Swap/Providers/JupiterTokensRepository.swift

```diff
@@ -3,42 +3,58 @@ import Combine
 import SolanaSwift
 import Resolver
 
+/// Tokens data returned by Jupiter
 struct JupiterTokensData {
+    /// List of Jupiter's supported tokens
     let tokens: [SwapToken]
+    /// Wallets owned by user
     let userWallets: [Wallet]
 }
 
+/// Repository that handle JupiterTokens
 protocol JupiterTokensRepository {
-    var status: AnyPublisher<JupiterDataStatus, Never> { get }
-
+    /// Current status of repository
+    var statusPublisher: AnyPublisher<JupiterDataStatus, Never> { get }
+    
+    /// Load repository
     func load() async
 }
 
+/// Status of current JupiterData
 enum JupiterDataStatus {
     case initial
     case loading
     case ready(swapTokens: [SwapToken], routeMap: RouteMap)
     case failed
 }
 
+/// Default implementaion of JupiterTokensRepository
 final class JupiterTokensRepositoryImpl: JupiterTokensRepository {
-
-    var status: AnyPublisher<JupiterDataStatus, Never> {
-        statusSubject.eraseToAnyPublisher()
-    }
     // MARK: - Dependencies
+
     private let jupiterClient: JupiterAPI
     private let localProvider: JupiterTokensProvider
     @Injected private var walletsRepository: WalletsRepository
 
-    // MARK: - Private params
+    // MARK: - Subjects
+
     private var statusSubject = CurrentValueSubject<JupiterDataStatus, Never>(.initial)
+    
+    // MARK: - Properties
+
+    var statusPublisher: AnyPublisher<JupiterDataStatus, Never> {
+        statusSubject.eraseToAnyPublisher()
+    }
+
+    // MARK: - Initializer
```

Comment:
I think that separating to more MARKers like this may help us indicating code a little bit better